### PR TITLE
[Keyboard]Add keyboard Setta21 / Nafuda

### PR DESCRIPTION
**Pull request by Salicylic-acid3**
Add default map for keyboard Setta21/rev1 and Nafuda.
Add Setta21 to the exclusion list.
Salicylic-acid3

### CODE REVIEW

File: public/keymaps/nafuda_default .json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "keyboard": "naked60/rev1",
```
Comment (noroadsleft) [inline public/keymaps/nafuda_default .json]:
```suggestion
  "keyboard": "nafuda",
```

File: public/keymaps/nafuda_default .json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  "keymap": "default_ad36bb3",
+  "commit": "687a24f2980e6e490ac8e43104f4176b033c9243",
+  "layout": "LAYOUT",
+  "layers": [
+    [
+                          "TG(1)",
+"LT(3, KC_BSPC)",         "KC_UP",     "TG(2)",
```
Comment (noroadsleft) [inline public/keymaps/nafuda_default .json]: `LT()` keycodes don't import correctly if there are spaces in the value.

```suggestion
 "LT(3,KC_BSPC)",         "KC_UP",     "TG(2)",
```

File: public/keymaps/nafuda_default .json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+       "KC_LEFT",       "KC_DOWN",   "KC_RIGHT"
```
Comment (noroadsleft) [inline public/keymaps/nafuda_default .json]: Configurator almost entirely uses the seven-character-maximum keycodes for its polished rendering mode.

```suggestion
       "KC_LEFT",       "KC_DOWN",   "KC_RGHT"
```

File: public/keymaps/nafuda_default .json [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    "LCTL(LSFT(KC_T))", "KC_WH_U",     "TG(2)",
+  "LCTL(LSFT(KC_TAB))", "KC_WH_D", "LCTL(KC_TAB)"
+    ],
+    [
+                        "RGB_VAD",
+          "MO(3)",      "RGB_SAD",   "RGB_VAI",
+        "RGB_MOD",      "RGB_TOG",   "RGB_SAI"
+    ]
+  ]
+}
```
Comment (noroadsleft) [inline public/keymaps/nafuda_default .json]: And after you've done all of :point_up: , please remove the space in the filename.

```
git mv "public/keymaps/nafuda_default .json" public/keymaps/nafuda_default.json
```

File: public/keymaps/setta21_rev1_default.json [hunk 5]
```diff
@@ -0,0 +1,32 @@
+{
+  "keyboard": "setta21/rev1",
+  "keymap": "default_12ea2e3",
+  "commit": "687a24f2980e6e490ac8e43104f4176b033c9243",
+  "layout": "LAYOUT",
```
Comment (noroadsleft) [inline public/keymaps/setta21_rev1_default.json]:
```suggestion
  "layout": "LAYOUT_numpad_6x4",
```

File: public/keymaps/setta21_rev1_default.json [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+  "layers": [
+    [
+ "LT(2,KC_ESC)",   "KC_F2",  "KC_EQL",  "KC_DEL",
+      "KC_NLCK", "KC_PSLS", "KC_PAST", "KC_PMNS",
+        "KC_P7",   "KC_P8",   "KC_P9",
+        "KC_P4",   "KC_P5",   "KC_P6", "KC_PPLS",
+        "KC_P1",   "KC_P2",   "KC_P2",
```
Comment (noroadsleft) [inline public/keymaps/setta21_rev1_default.json]:
```suggestion
        "KC_P1",   "KC_P2",   "KC_P3",
```

File: public/keymaps/setta21_rev1_default.json [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+ "LT(1, KC_P0)",            "KC_PDOT", "KC_PENT"
```
Comment (noroadsleft) [inline public/keymaps/setta21_rev1_default.json]:
```suggestion
  "LT(1,KC_P0)",            "KC_PDOT", "KC_PENT"
```

File: public/keymaps/setta21_rev1_default.json [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    [
+       "KC_ESC", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+        "KC_NO", "KC_TRNS", "KC_TRNS", "KC_TRNS",
+        "KC_NO",   "KC_UP",   "KC_NO",
+      "KC_LEFT", "KC_DOWN","KC_RIGHT", "KC_TRNS",
```
Comment (noroadsleft) [inline public/keymaps/setta21_rev1_default.json]:
```suggestion
      "KC_LEFT", "KC_DOWN", "KC_RGHT", "KC_TRNS",
```

File: public/keymaps/nafuda_default .json (hunk 4 above)
Comment (Salicylic-acid3) [reply public/keymaps/nafuda_default .json]: Thank you for pointing out.

**Comment by noroadsleft** [comment]
Excellent, thanks!

**Pull request merged**